lib/pbench/server/db/database.py

```diff
@@ -0,0 +1,54 @@
+import sys
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from pbench.server import NoOptionError, NoSectionError
+
+
+class Database:
+    """ Database - a base class for declaring SQLAlchemy database model
+    classes.
+    """
+
+    Base = declarative_base()
+    # Initialize the db scoped session
+    db_session = scoped_session(sessionmaker(autocommit=False, autoflush=False))
+
+    @staticmethod
+    def get_engine_uri(config, logger):
+        try:
+            psql = config.get("Postgres", "db_uri")
+            return psql
+        except (NoSectionError, NoOptionError):
+            if logger:
+                logger.error(
+                    "Failed to find an [Postgres] section" " in configuration file."
```

Comment:
Again, this entire file is stolen from #1937, and I hadn't really paid attention. In this case, however, since you pointed it out, it bugs me enough that I'm changing it anyway.